README.md

```diff
@@ -54,30 +56,37 @@ The pre-requisites for using this image are:
     * Needs at least write access for `pull-requests` scope - see [documentation][scopes]
   * Can be a personal access token (PAT) - see [documentation][PAT]
     * Needs the `repo` scope or minimally the `public_repo` scope if private repositories are not used
-* A [Phylum token](https://docs.phylum.io/docs/api-keys) with API access
-  * [Contact Phylum](https://phylum.io/contact-us/) or create an account and register to gain access
-    * See also [`phylum auth register`](https://docs.phylum.io/docs/phylum_auth_register) command documentation
+* A [Phylum token][phylum_tokens] with API access
+  * [Contact Phylum][phylum_contact] or [register][app_register] to gain access
+    * See also [`phylum auth register`][phylum_register] command documentation
   * Consider using a bot or group account for this token
 * Access to the Phylum API endpoints
   * That usually means a connection to the internet, optionally via a proxy
   * Support for on-premises installs are not available at this time
 * A `.phylum_project` file exists at the root of the repository
-  * See [`phylum project`](https://docs.phylum.io/docs/phylum_project) and
-    [`phylum project create`](https://docs.phylum.io/docs/phylum_project_create) command documentation
+  * See [`phylum project`][phylum_project] and
+    [`phylum project create`][phylum_project_create] command documentation
 
 [container]: https://docs.github.com/en/actions/creating-actions/creating-a-docker-container-action
 [package]: https://github.com/phylum-dev/phylum-ci/pkgs/container/phylum-ci
 [gh_token]: https://docs.github.com/en/actions/security-guides/automatic-token-authentication
 [scopes]: https://docs.github.com/en/developers/apps/building-oauth-apps/scopes-for-oauth-apps#available-scopes
 [PAT]: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token
+[phylum_contact]: https://phylum.io/contact-us
+[app_register]: https://app.phylum.io/register
+[phylum_tokens]: https://docs.phylum.io/docs/api-keys
+[phylum_register]: https://docs.phylum.io/docs/phylum_auth_register
+[phylum_project]: https://docs.phylum.io/docs/phylum_project
+[phylum_project_create]: https://docs.phylum.io/docs/phylum_project_create
```

Comment:
As we use this style of link more and more, I am starting to think that we need a markdown linting tool to check for small typos and extra links, etc...

This all looks fine to my eye, but who knows if I've actually given it a detailed enough look...

Reply:
Agreed. There are three that I use for markdown:
* VS Code itself (markdown support [is provided natively now](https://code.visualstudio.com/blogs/2022/08/16/markdown-language-server) via a language server)
* Markdown All in One extension for VS Code
  * This is the one that tells me when the link references are bad
  * It is also good for viewing previews directly in VS Code
* markdownlint
  * This one is more a pure linting tool "rules checker"